### DISCUSSION TITLE
How can I get all messages and reactions on them from one discord channel using discord.py?

### DISCUSSION THREAD

**Discussion by CYBEROUT-me** [Rapptz/discord.py#8268] [Q&A]
I'm trying to create bot, that will give a role for user, whose message will have the most reactions. How can I do this? 
I assume for this it will be necessary to upload messages to the database.

**Comment by Tumpa-Prizrak** [comment]
You can use [history](https://discordpy.readthedocs.io/en/stable/api.html?highlight=text#discord.TextChannel.history) and [reactions](https://discordpy.readthedocs.io/en/stable/api.html?highlight=message#discord.Message.reactions) for this
